code/controllers/UserController.php

```diff
@@ -17,8 +17,9 @@ public function index()
     {
         if (!Member::currentUserID()) {
             return $this->redirect('/Security/login');
-        }
-        return $this->renderWith(array('UserController', 'Page'));
+        } else if (UserInvitation::create()->canCreate(Member::CurrentUser()) == false){
```

Comment:
My understanding is that when Silverstripe receives the the form submission it treats it as a plain POST request, that means that the form action also needs to be secure. Hence some type of permission check is needed is needed in `sendInvite`. How about if the check on `canCreate` is replaced with `Permission::check('ACCESS_USER_INVITATIONS')`?